db.json

```diff
@@ -212,5 +212,5 @@
     }
   ],
   "cart": [],
-  "orders": []
+  "feedbacks": []
```

Comment:
you dont need to delete the field itself, just clear its content before pushing.
we dont need the cart, orders, feedbacks arrays (JSON server endpoints) to be populated with unnecessary data. These fields' content are dinamically added, updated, removed upon using our application.